pkg/kubernetes/templates/v1beta2/kubeadm_config.go

```diff
@@ -187,8 +187,10 @@ func GetKubeletConfiguration(runtime connector.Runtime, kubeConf *common.KubeCon
 			"cpu":    "200m",
 			"memory": "250Mi",
 		},
+		"podPidsLimit": 1000,
 		"evictionHard": map[string]string{
 			"memory.available": "5%",
+			"pid.available": "10%",
```

Comment:
Are there any potential issues if keep using the default value?

Reply:
In some cluster, because the pod takes up a large amount of pid, the pid of nodes is exhausted, which leads to abnormal cluster. This kind of resource limiting helps to prevent simple fork bombs from affecting operation of an entire cluster. 
These parameters are @vincenthe11 accumulated experience in clusters operation.